lookout/style/format/virtual_node.py

```diff
@@ -1,10 +1,11 @@
 """Defines VirtualNode - a class which backs any source code token."""
 from typing import Callable, Iterable, Mapping, NamedTuple, Optional, Set, Tuple, Union
 
-import bblfsh
+from bblfsh import Node
```

Comment:
I would rather use `bblfsh.Node` everywhere to reduce the confusion between `AnyNode`, `VirtualNode` and `Node`

Reply:
I'll import as bblfsh_Node then if it's fine for you to stay consistent with the import style we prefer in this repo